blockstack/blockstack-browser
# #1193 Enable subdomain user registration

Completes #1193 

![subdomain-registration](https://user-images.githubusercontent.com/4405356/35883037-f6b18638-0b53-11e8-95ed-191b3dacf407.gif)

To test code:
1. Run `npm install` to install Enzyme and its dependencies for React component testing
2. Verify tests pass using `npm run test`

To test UI:
1. Navigate to `http://localhost:3000/profiles/i/add-username/0/search` in development mode
2. Enter a username (if one does not exist)
3. See that the app searches through the subdomain records and shows that you can register a subdomain with your username.
